### DISCUSSION TITLE
make real periodicals [½ SOLUTION use USB]

### DISCUSSION THREAD

**Discussion by mmagnus** [mmagnus/Pocket-Plus-Calibre-Plugin#41] [Ideas]
I would love to be able to group all Pocket ebooks under one Pocket periodical icon (where all ebooks are grouped). But I have no idea how to do it, any ideas?

https://stackoverflow.com/questions/5379565/kindle-periodical-format

> Amazon automatically converts all mobi files sent by email into personal documents. You have to use USB if you want them to remain periodicals on the Kindle.

http://www.mobileread.mobi/forums/showthread.php?t=241787

my research here....

**Comment by mmagnus** [comment]
OK, it's super crazy


**MOBI VS POBI**
When you put a MOBI file of a periodical, it will have .MOBI extension for now, but when you reconnect Kindle it the file will have .pobi extension now. POBI is a variation of MOBI and enables the display of articles and sections in periodical content.

**AUTO-COPY TO KINDLE**
If you download a News book it will be automatically copied to your device (if connected), at least by default, to the folder News on your device. This is pretty cool, but yet, if you don't know about it, this drives you crazy when you see files appearing there for no obvious reason. It's good to know this is how it works.

**COPY MOBI TO KINDLE**
If you copy a MOBI file, you will see no cover, but you will get Issues. It seems that for MOBI Amazon downloads cover on its own, so if you put there your book from some different source there will be no cover there. Maybe there is some one to fix it, I'm not sure...

**BOOKS NOT SHOWED**
To make it even more crazy, sometimes when you copy files to Kindle you will not see them. You might reset your Kindle and wait till the indexing program will index your books. This also made my testing crazy ...
https://manual.calibre-ebook.com/faq.html#id31

**OK, I MADE IT!**
Funny, at the very end of this day, I downloaded one news with Calibre and it was copied to Kindle and it was there as fully working (fully = no cover) periodical. Grr.... 

**COPY VS SEND VIA MAIL**

Amazon automatically converts all MOBI files (re-formats files to remove periodical
metadata from the header) sent by email into personal documents. You have to use USB if you want them to remain periodicals on the Kindle.

**MORE ON COVERS**
Pick what you prefer, a file being a Book or a Personal file.
https://manual.calibre-ebook.com/faq.html#the-covers-for-my-mobi-files-have-stopped-showing-up-in-kindle-for-pc-kindle-for-android-ipad-etc
no more is your created MOBI should be Personal file or Book, add/remove '[PDOC]' in Preference -> Conversion Section -> Output Options -> MOBI Output
https://www.reddit.com/r/kindle/comments/1gjjjd/calibre_kindle_paperwhite_periodicals_sometimes/

![screenshot_2020_05_18T13_05_10+0200](https://user-images.githubusercontent.com/118740/82208520-f9089280-990b-11ea-8fdc-b5efc9d45fa0.png)
Figure. Issue (via USB) vs "Covered" Pocket over Amazon Cloud.

**Comment by Monirzadeh** [comment]
i don't get what do you want exactly to do.
i use an old version of kindle firmware and use collection to group my pocket book under specific collection with [this plugin](https://www.mobileread.com/forums/showthread.php?t=244202)
is it a new feature that i miss?

**Comment by mmagnus** [comment]
Ideally I wanted to have Issues like for journals that you buy from Amazon. A collection of nicely files with covers.. but it seems that you can't get really that on your own.

Yeah, collections maybe is the way to go, thanks for this, I will take a look at that.

![screenshot_2020_05_19T21_46_14+0200](https://user-images.githubusercontent.com/118740/82371418-728ea680-9a1a-11ea-8119-dd25be1792c2.png)
![screenshot_2020_05_19T21_46_17+0200](https://user-images.githubusercontent.com/118740/82371498-9a7e0a00-9a1a-11ea-8ca7-38830fefc87f.png)

**Comment by Monirzadeh** [comment]
sound good.
it is not completely related, but there is an [awesome project](https://github.com/koreader/koreader).  maybe can develop a Pocket Plugin for that. (i read most of my books with that )

**Comment by mmagnus** [comment]
OK, Calibre, Release: 4.17 [23 May, 2020], has something interesting to help get nice covers via USB, worked for me, for News.

Add a workaround for Amazon destroying cover thumbnails on e-ink Kindles.
For books sent with this release of calibre, reconnecting the Kindle to calibre after the cover has been destroyed will restore it automatically
